src/main/java/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import java.util.List;
+
+public interface ReportService {
+    List<String> creatingReport();
```

Comment:
имена методов - существительные 
```suggestion
    List<String> createReport();
```